### PR TITLE
Implement #1

### PR DESCRIPTION
**Pull request by Valeri0p**
Related to #1

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -3,20 +3,52 @@
...
+    # Building the command. Ex: "ping -c 1 google.com"
+    command = ['ping', param, '1', host]
+
+    return subprocess.call(command) == 0
+
+
 async def make_request_to_telegram(service_name: str) -> dict:
-    message = f'⚠️ The service {service_name} results offline!'
+    
+    if check_ok(service_name) == False or myping(service_name[7:-1]) == False:
```
Comment (Valeri0p) [inline src/main.py]: ping only understands hostnames and IPs, not urls. url stripping only works with "http://"
They both aren't very reliable methods to know if a service is online or not, right now.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,6 @@ Just set two env variables:
 ### Example in bash
 ```bash
 export QDBotToken="123456:ABC-DEF1234ghIkl-zyx57W2v1u123ew11" # Your bot token
-export QDBotIDs=10000000                                      # Single ID
-export QDBotIDs=10000000;10000001;10000002                    # Multiple IDs
+export QDBotIDs="10000000"                                      # Single ID
```
Comment (domenicoblanco) [inline README.md:18]: Not necessary double quotes

File: src/main.py [hunk 3]
```diff
@@ -3,20 +3,52 @@
...
+import platform    # For getting the operating system name
+import subprocess  # For executing a shell command
 
 def get_users() -> list[str]:
+    #print(getenv('QDBotIDs'))
     return getenv('QDBotIDs').split(';')
 
+def check_ok(url: str) -> bool:
+    r = httpx.get(url)
+    #there are various status codes, duckduckgo returns 301; is there a keyword only for errors?
```
Comment (domenicoblanco) [inline src/main.py:15]: Normally a 200 is a fulfilled request, I think that we could check if the error code is greater than 299

File: src/main.py [hunk 4]
```diff
@@ -3,20 +3,52 @@
...
+    #return r.status_code == httpx.codes.OK
+    return r.status_code == 301
+
+#si, è copiata
+def myping(host):
```
Comment (domenicoblanco) [inline src/main.py]: This doesn't follow the snake_case

File: src/main.py (hunk 1 above)
Comment (domenicoblanco) [reply src/main.py]: We could use two separate keys for address that will be pinged or will receive a GET request

**Review by domenicoblanco — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Valeri0p) [reply README.md:18]: Row 17 uses double quotes, let's keep it for consistency.

File: src/main.py (hunk 3 above)
Comment (Valeri0p) [reply src/main.py:15]: Since redirects (3xx) don't mean a service is offline, I set it to <400

**Review by domenicoblanco — APPROVED** [review]

**Comment by Valeri0p** [comment]
N.B.: using example.org as it's better suited for a didactic project.

**Pull request merged**